docs/plugins/pluggable_rule_engine.md

```diff
@@ -92,48 +90,52 @@ Consider the following `server_config.json` with the accompanying `example.re`,
 }
 ```
 
+#### core.py
 ```python
-# core.py
+RULE_ENGINE_CONTINUE = 5000000
+
 def pep_resource_close_pre(rule_args, callback, rei):
-    callback.writeLine("serverLog","XXXX - core.py")
-```
+    callback.writeLine("serverLog", "XXXX - core.py")
 
-```
-# example.re
-pep_resource_close_pre(*INSTANCE, *CONTEXT, *OUT){
-    writeLine("serverLog","XXXX - example.re")
-}
+    # Instruct the REPF to let the next rule engine plugin handle this PEP.
+    # If we remove the line below, the PEP in core.re will not execute.
+    return RULE_ENGINE_CONTINUE
 ```
 
+#### core.re
 ```
-# core.re
-pep_resource_close_pre(*INSTANCE, *CONTEXT, *OUT){
-    writeLine("serverLog","XXXX - core.re top")
+RULE_ENGINE_CONTINUE = 5000000
+
+pep_resource_close_pre(*INSTANCE, *CONTEXT, *OUT) {
+    writeLine("serverLog", "XXXX - core.re - first")
+
+    # Instruct the REPF to let the next rule engine plugin handle this PEP.
+    RULE_ENGINE_CONTINUE
 }
 
-pep_resource_close_pre(*INSTANCE, *CONTEXT, *OUT){
-    writeLine("serverLog","XXXX - core.re bottom")
+# This rule will never execute because it shadows the rule above.
+# Only the first rule in the scope of the iRODS Rule Language will execute.
+pep_resource_close_pre(*INSTANCE, *CONTEXT, *OUT) {
+    writeLine("serverLog", "XXXX - core.re - second")
 }
 ```
 
+When `pep_resource_close_pre` is triggered, both implementations for this PEP will execute. The implementation in `core.py` will execute first, followed by the implementation in `core.re`.
```

Comment:
"both implementations" -> "two of the three implementations"

"the implementation in `core.re`" -> "the first implementation in `core.re`"

Reply:
Done.